README.md

```diff
@@ -157,6 +157,7 @@ If you believe you've found an issue with this tool, please follow the ["Reporti
 | Greek (Ελληνικά)                            | [sofronas](https://github.com/sofronas)                                                      | UI                                    |
 | Hebrew (עברית)                              | [ronbentata](https://github.com/ronbentata)                                                  | UI and [README](./README-Hebrew.md)   |
 | Hindi (हिन्दी)                              | [triach-rold](https://github.com/triach-rold)                                                | UI and [README](./README-Hindi.md)    |
+| Hungarian (Magyar)                            | [synexdev01](https://github.com/synexdev01)                                                    | UI                                    |
```

Comment:
The formatting checker failed on this - you need to run prettier on this file too